<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSearchDevices">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Search devices</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbKeys1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Keys 1 (D9:D7:88:35:C4:F8)</string>
    </property>
    <widget class="QPushButton" name="btnKeys1_mid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Mid Alert(0x01)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKeys1_high">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>31</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>High Alert(0x02)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKeys1_bat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Battery level (0x2a19)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneKeys1_bat">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbKeys2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Keys 2 (E0:0E:F7:E4:6B:14)</string>
    </property>
    <widget class="QPushButton" name="btnKeys2_mid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Mid Alert(0x01)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKeys2_high">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>High Alert(0x02)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKeys2_bat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Battery level (0x2a19)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lneKeys2_bat">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbPer1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Peripherial 1  (C1:44:EA:74:47:42)</string>
    </property>
    <widget class="QPushButton" name="btnPer1_temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Temperature (0x2a6e)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer1_temp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>29</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer1_dig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>89</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_dig_read">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Digital (0x2a56)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_light">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Light Intensity (custom)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer1_light">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_hum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Humidity (custom)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer1_hum">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer1_alert">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>329</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_alert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>329</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
New Alert (2a46)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_led_on">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>389</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LED On (custom)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_led_off">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LED Off (custom)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer1_but">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>449</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_but">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>449</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Button (custom)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_dig_write">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>149</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Write
Digital (0x2a56)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer1_read_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Read all characteristics</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grbPer2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Peripherial 2  (C5:03:B9:81:0B:D8)</string>
    </property>
    <widget class="QPushButton" name="btnPer2_temp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Temperature (0x2a6e)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer2_temp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer2_dig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_dig_read">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>91</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Digital (0x2a56)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_light">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Light Intensity (custom)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer2_light">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_hum">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Humidity (custom)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer2_hum">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer2_alert">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>329</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_alert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>329</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
New Alert (2a46)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_led_on">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>389</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LED On (custom)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_led_off">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>390</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>LED Off (custom)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnePer2_but">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>449</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_but">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>449</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Read
Button (custom)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_dig_write">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Write
Digital (0x2a56)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPer2_read_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Read all characteristics</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="chbclosing_doors">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed door if least than two keys finded</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
